src/Microsoft.IdentityModel.Tokens/InternalValidators.cs

```diff
@@ -46,7 +46,7 @@ internal static class InternalValidators
         /// <exception cref="SecurityTokenUnableToValidateException">
         /// If the lifetime or issuer are invalid
         /// </exception>
-        internal static void ValidateLifetimeAndIssuerAfterSignatureNotValidatedJwt(SecurityToken securityToken, DateTime? notBefore, DateTime? expires, string kid, TokenValidationParameters validationParameters, BaseConfiguration configuration, StringBuilder exceptionStrings)
+        internal static void ValidateLifetimeAndIssuerAfterSignatureNotValidatedJwt(SecurityToken securityToken, DateTime? notBefore, DateTime? expires, string kid, TokenValidationParameters validationParameters, BaseConfiguration configuration, StringBuilder exceptionStrings, int numKeysInConfiguration, int numKeysInTokenValidationParameters)
```

Comment:
consider stacking parameters as above.

Reply:
Fixed! https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1786/commits/27f73791bb1e017be2e7bc1a0b4c25cba31b9a78